storage/series/series.go

```diff
@@ -220,6 +231,13 @@ func (s *dbSeries) IsEmpty() bool {
 	return false
 }
 
+func (s *dbSeries) NumActiveBlocks() int {
+	s.RLock()
+	value := s.blocks.Len() + s.buffer.Stats().wiredBlocks
```

Comment:
can you refresh me on active/wired/open?

Confused as to why we would be adding these up here

Reply:
So Active == wired + unwired
Open == wired (but only mutable wired blocks)